README.rst

```diff
@@ -66,6 +66,16 @@ ecosystem of existing plugins for popular frameworks such as `pylint-django`_ or
 .. _`pylint-django`: https://github.com/PyCQA/pylint-django
 .. _`pylint-sonarjson`: https://github.com/omegacen/pylint-sonarjson
 
+.. note::
+
+    Pylint supports the Python standard library out of the box. Third-party
+    libraries are not always supported, so a plugin might be needed. A good place
+    to start is ``PyPI`` which often returns a plugin by searching for
+    ``pylint <library>``. `pylint-django`_ and `pylint-pydantic`_ are examples
+    of such plugins.
+
+.. _`pylint-pydantic`: https://pypi.org/project/pylint-pydantic
+
```

Comment:
Perhaps we should also link to plugin usage so that a user understands how to use the plugins.

Reply:
The Readme also contains plugin information in the preceding paragraph; lets put them together?

Reply:
Yeah I wasn't sure how to combine those. Shall we make one `note`? Or one main paragraph?

Reply:
I was thinking of converting your `note` paragraph to be `plugins` then add the existing plugin info here + your new content. Happy to go another way if you prefer also.